### PR TITLE
Spanish translation

### PR DESCRIPTION
**Pull request by Zybax**
I changed some of the words and phrasing to make it more appealing to hispanic readers, this way it doesnt sound like it was translated but natively written on spanish

### CODE REVIEW

**Comment by js-kyle** [comment]
@Zybax looks great, thank you! I will leave it here for a short time to see if any of our other Spanish contributors can review, otherwise it looks good

**Comment by js-kyle** [comment]
@all-contributors-bot please add @Zybax for translation

File: README.spanish.md [hunk 1]
```diff
@@ -17,12 +17,12 @@
  [![nodepractices](/assets/images/twitter-s.png)](https://twitter.com/nodepractices/) **¡Síguenos en Twitter!** [**@nodepractices**](https://twitter.com/nodepractices/)
  <br/>
 
-# ¡Bienvenido! 3 cosas que necesitas saber primero:
-**1. Cuando lees aquí, lees docenas de los mejores artículos de Node.JS -** este es un resumen y conservación del contenido mejor clasificado de las mejores prácticas de NodeJS
+# ¡Bienvenido! 3 cosas que deberias saber primero:
+**1. Aquí encontraras docenas de los mejores artículos sobre Node.JS -** este es un resumen y conservación del contenido mejor clasificado de las mejores prácticas de NodeJS
```
Comment (Anonyma) [inline README.spanish.md]: Missing accent on "encontrarás"! Otherwise LGTM :smile:

Comment (Zybax) [reply README.spanish.md]: Got it, just fixed it, thanks

**Pull request merged**